### ISSUE TITLE
Help Request: Trouble Setting Up On Intellij Idea

### ISSUE THREAD

**Issue by nashmia-riaz** [TEAMMATES/teammates#8765]
I can't run the server in Intellij Idea.

- **Commit SHA:** 997ca701a2b19803457b1580c879fc117040fd42
- **Operating System:** Windows 10
- **JDK Version:** 1.8
- **Node.js Version:** v8.10.0
- **Google Cloud SDK Version:** 196.0.0
- **Eclipse / IntelliJ Version (if applicable):** Intellij Ultimate 2018.1

**What I was trying to do / What I need help with**
I am trying to set up the environment for Intellij Idea and get the NoSuchFileException error, saying that appengine-web.xml doesn't exist.

**Attempts to resolve the problem**
I've tried to setup the environment again several times now, following the tutorials listed [here](https://github.com/TEAMMATES/teammates/blob/master/docs/setting-up.md) and [here](https://github.com/TEAMMATES/teammates/blob/master/docs/ide-setup.md) but to no success.

**Comment by wkurniawan07** [comment]
> `appengine-web.xml` doesn't exist.

If it doesn't exist, that's a step in the setting up guide missed. Is it a problem where it actually exists but IntelliJ fails to recognize it?

**Comment by nashmia-riaz** [comment]
@wkurniawan07 I don't think I missed a step as I managed to fix the problem by downgrading my Intellij Idea to 2017.3.5.

**Comment by whipermr5** [comment]
@nashmia-riaz I'm using IntelliJ Ultimate 2018.1 too and it's working fine, though I set up the project on an earlier version. Did you confirm that `appengine-web.xml` existed when you were on 2018.1?

**Comment by nashmia-riaz** [comment]
@whipermr5 Apologies for not stating earlier. As far as I can tell, it wasn't being created on 2018.1. Only two folders (WEB-INF and META-INF) would get created in teammates.war.

**Comment by whipermr5** [comment]
So `src/main/webapp/WEB-INF/appengine-web.xml` existed but was not copied over? Also, now that you've gotten it working in 2017.3.5, does it work if you upgrade to 2018.1? That would help verify the problem.

**Comment by nashmia-riaz** [comment]
@whipermr5 yes, that is true. It does exist in src/main/webapp/WEB-INF/appengine-web.xml but is not copied over into the out folder. 

The problem persists. Here's what I know:
1. I can't set up a new environment on 2018.1. It tells me that appengine-web.xml doesn't exist in the out folder. However, I can set up that same environment in 2017.3.5 without any errors.
2. (This is trivial but) I can run an environment I have already set up in 2017.3.5 using 2018.1.

Let me know what else I can do to help figure this out.

**Comment by whipermr5** [comment]
@nashmia-riaz Thanks for the report! I was able to reproduce the issue upon performing a fresh setup on IntelliJ IDEA 2018.1:
<img width="990" alt="screen shot 2018-04-16 at 2 13 34 pm" src="https://user-images.githubusercontent.com/6811428/38793350-816c00e0-4183-11e8-8d67-45b6b4326292.png">

**Comment by whipermr5** [comment]
Somehow the exploded war folder only contains these:
<img width="224" alt="screen shot 2018-04-16 at 2 39 11 pm" src="https://user-images.githubusercontent.com/6811428/38793489-0b4416ae-4184-11e8-97f4-78f764291e1b.png">

The expected behaviour is that it should contain all these:
<img width="252" alt="screen shot 2018-04-16 at 2 40 42 pm" src="https://user-images.githubusercontent.com/6811428/38793531-4083b4f0-4184-11e8-87e3-3ab2b5cd4eb3.png">

**Comment by nashmia-riaz** [comment]
@whipermr5 this is exactly the problem I faced. 👍

**Comment by whipermr5** [comment]
The problem seems to be fixed in 2018.1.1 (it was probably [this issue](https://youtrack.jetbrains.com/issue/IDEA-189112)). @nashmia-riaz can verify?

**Comment by nashmia-riaz** [comment]
Yes, I can confirm. Local server is up and running now.
